pkg/cmd/pr/merge/merge_test.go

```diff
@@ -279,17 +279,28 @@ func TestPrMerge(t *testing.T) {
 		baseRepo("OWNER", "REPO", "master"),
 	)
 
+	http.Register(
```

Comment:
these new stubs are needed because of the addition of the call to `api.GitHubRepoWithMergeQueue(apiClient, baseRepo, pr.BaseRefName)` in `NewMergeContext` to find out if the PR is already in the merge queue. It is a bit repetitive, not sure if this is worth extracting or having each test be self-contained.

I also noticed that this is sometimes called `http` and sometimes called `tr` depending on the test, I didn't rename them but think it would be good to do for consistency.

Reply:
It also means that the argument to `baseRepo` in `RunCommandFinder` is somewhat redundant, as mentioned in `NewMergeContext` above, which looks like a code smell.